packages/v4/patternfly-docs/content/design-guidelines/components/alert/alert.md

```diff
@@ -5,68 +5,67 @@ related: ['Alert', 'Alert group']
- 
-### Plain inline alerts
-Plain inline alerts are basic variations of inline alerts. They communicate information about a temporarily persistent error or other activity. A plain inline alert is less visually intrusive than a bordered inline alert, and is designed to be placed in content-heavy areas, such as within a form, wizard, or drawer. This alert type persists until the error or action is resolved.
- 
-![plain inline notification placement](./img/plain-inline-notification.png)
- 
-## Toast alerts
-Toast alerts communicate information about an update, confirmation, or other activity. A toast alert overlays content in the upper-right corner of the page and disappears when it times out or when the user dismisses it.
- 
-![toast notification placement](./img/toast-notification.png)
- 
-## Elements
-Toast alerts are always dismissible, but bordered inline alerts can be both dismissable and non-dismissible. All other elements are consistent between toast and bordered inline alerts.
+## Elements: 
+Elements vary depending on the variation of Alert. Toast alerts are always dismissible, but bordered inline alerts can be both dismissable and non-dismissible. All other elements are consistent between toast and bordered inline alerts.
 Plain inline alerts are never dismissable, and consist of only a status area and alert title.
  
 <img src="./img/alert-types.png" alt="Alert types" width="868" />
- 
-1. **Status area**: The status area indicates the severity of the alert with a specific icon and color.  
-2. **Alert title**: The alert title concisely communicates the alert’s main message. Its text color is coded to match the alert type. 
-3. **Close button**: The close button is used to dismiss the alert. It’s required for toast alerts and optional for inline alerts. 
-4. **Actions (optional)**: The actions area enables the user to take a specific action in relation to the alert. 1 or more actions can be placed inside of an alert. 
+
+1. **Status area**: The status area indicates the severity of the alert with a specific icon and color. 
+
+2. **Alert title**: The alert title concisely communicates the alert’s main message. Its text color is coded to match the alert type.
+
+3. **Close button**: The close button is used to dismiss the alert. It’s required for toast alerts and optional for inline alerts.
+
+4. **Actions (optional)**: The actions area enables the user to take a specific action in relation to the alert. 1 or more actions can be placed inside of an alert.
+
 5. **Description (optional)**: The description field communicates additional information to the user. It can also include a navigation link as an alternative to an action.
+
 6. **Expandable alert (optional)**: The expansion reveals additional information.
- 
-## Communicating severity
+### Communicating severity
 The status icon, background, and alert title are coded to communicate the severity of an alert.
- 
 <img src="./img/alert-status.png" alt="alert styles" width="700" />
- 
```

Comment:
Also you need to add a line of space between the image and the table - that might also be causing the table not to show up properly because it "starts" on the same line as the image rn

<img width="904" alt="image" src="https://user-images.githubusercontent.com/51165119/178366671-9ddafc76-8c41-431d-ace4-7f1c4c194282.png">